### ISSUE TITLE
Error Message

### ISSUE THREAD

**Issue by elliott2081** [FMCorz/moodle-repository_evernote#2]
I have downloaded and installed the plugin, set up an API key and it seems to be working well, except I can only link to an file, I can't make a copy. It gives the error 
[ERROR]
"can't download file"

![cannotdownloadfile](https://f.cloud.github.com/assets/566244/1457465/d8b05b6c-4340-11e3-8a24-7f1d91321ccc.png)

FROM THE APACHE ERROR LOG

[Fri Nov 01 22:17:49 2013] [error] [client .........] Default exception handler: Cannot download this file Debug: \nError code: cannotdownload\n\* line 664 of /repository/evernote/lib.php: repository_exception thrown\n\* line 298 of /repository/repository_ajax.php: call to repository_evernote->get_file()\n, referer: https://............/course/modedit.php?add=resource&type=&course=13&section=0&return=0&sr=0

**Comment by FMCorz** [comment]
Thanks for reporting this bug.

I could not reproduce this bug, could you confirm what version of Moodle and repository_evernote you are using? The release 1.2.1 seems to be fixed, though if you are part of the few people who downloaded the version 1.2.0 then you will experience this problem.

Please let me know so I can fix that ASAP if it is still an issue.

Cheers,
Fred

**Comment by FMCorz** [comment]
Off topic. I am really curious, what Moodle theme is this (on your screenshot)? :-)

**Comment by elliott2081** [comment]
I'm wondering if it's a problem with evernote permissions. It seems strange though, because it lets me log in and I can see my data. However, I realized that I'm on my dev server and its not the web site that I registered so that might be part of it. I will be investigating more this weekend and will update you. I downloaded the plugin 3 days ago so I might have 1.2.0. I'll check tomorrow. 

It's a custom theme. We're a Mahoodle + a fair amount of other custom code as well. The main guy is really good at design and made it the theme with bootstrap and it works really well on different devices.
